### ISSUE TITLE
Some commit messages are not displayed in commit log table

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#173]
```
What steps will reproduce the problem?
1. Open repository (for example, 
https://include-what-you-use.googlecode.com/svn/trunk/)
2. Observe revision 345 in commit log table.

What is the expected output? What do you see instead?
Expect first row of commit message to be displayed in table. Instead 'Message' 
column for this row is empty. As you can see, it is not the only one revision 
with such a problem.

What version of the product are you using? On what operating system?
svnX 1.3.3 on Mac OS X 10.6.8.
```

Original issue reported on code.google.com by `vsap...@gmail.com` on 10 Jun 2012 at 1:42

**Comment by GoogleCodeExporter** [comment]
```
Actually, svnX is displaying the first line of the commit messages.
However, the first line of the commit messages in question is empty, which 
makes them appear blank in the list.

I have now added a filter that trims leading a trailing white-space from commit 
messages before displaying them in the repo log list.
[This will appear in a future release of svnX.]
Thanks for your report.
```

Original comment by `chris...@gmail.com` on 10 Jun 2012 at 5:26
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svnX 1.3.4.
```

Original comment by `chris...@gmail.com` on 4 Sep 2012 at 5:39
- Changed state: **Fixed**
